Userland/Libraries/LibGfx/Font/OpenType/Tables.h

```diff
@@ -219,6 +211,41 @@ class OS2 {
     }
 
 private:
+    struct Version0 {
+        BigEndian<u16> version;
+        BigEndian<i16> avg_char_width;
+        BigEndian<u16> us_weight_class;
+        BigEndian<u16> us_width_class;
+        BigEndian<u16> fs_type;
+        BigEndian<i16> y_subscript_x_size;
+        BigEndian<i16> y_subscript_y_size;
+        BigEndian<i16> y_subscript_x_offset;
+        BigEndian<i16> y_subscript_y_offset;
+        BigEndian<i16> y_superscript_x_size;
+        BigEndian<i16> y_superscript_y_size;
+        BigEndian<i16> y_superscript_x_offset;
+        BigEndian<i16> y_superscript_y_offset;
+        BigEndian<i16> y_strikeout_size;
+        BigEndian<i16> y_strikeout_position;
+        BigEndian<i16> s_family_class;
+        u8 panose[10];
+        BigEndian<u32> ul_unicode_range1;
```

Comment:
Do we need to pack this struct tightly? If I counted correctly `ul_unicode_range1` starts at byte offset 42.

Reply:
`BigEndian` is packed at the class level, so there should be no alignment padding inserted:
```cpp
template<typename T>
class [[gnu::packed]] BigEndian {
```

Reply:
Oh didn't know that